src/utils/index.js

```diff
@@ -1,30 +1,73 @@
 import accepts from 'attr-accept'
 
+// Error codes
+export const FILE_INVALID_TYPE = 'file-invalid-type'
+export const FILE_TOO_LARGE = 'file-too-large'
+export const FILE_TOO_SMALL = 'file-too-small'
+export const FILE_EXCESSIVE = 'file-excessive'
```

Comment:
Let's bee more explicit:
```suggestion
export const TOO_MANY_FILES = 'too-many-files'
```